### PR TITLE
Change UoR to UOxf 

### PR DESCRIPTION
**Pull request by SarahAlidoost**
For more information about the method, see [Where can I find more information?](https://eucp-project.github.io/atlas/about)

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ per each `institute`-`method`:
 - [Preprocess ETHZ-ClimWIP data](cleanup_ETHZ_ClimWIP_atlas_netcdf.ipynb)
 - [Preprocess ICTP-REA data](cleanup_ICTP_REA_atlas_netcdf.ipynb)
 - [Preprocess UKMO-UKCP data](cleanup_UKMO_UKCP_atlas_netcdf.ipynb)
-- [Preprocess UoR-CALL data](cleanup_UoR_CALL_atlas_netcdf.ipynb)
+- [Preprocess Uoxf-CALL data](cleanup_Uoxf_CALL_atlas_netcdf.ipynb)
```
Comment (bvreede) [inline python/README.md:31]:
```suggestion
- [Preprocess UOxf-CALL data](cleanup_UOxf_CALL_atlas_netcdf.ipynb)
```

**Review by bvreede — CHANGES_REQUESTED** [review]
Good work! Just a small capitalization issue.

**Comment by SarahAlidoost** [comment]
> Good work! Just a small capitalization issue.

Thanks, done. :+1:

**Review by bvreede — APPROVED** [review]
👍🏽

**Pull request merged**